samples/amber.cc

```diff
@@ -268,6 +291,13 @@ int main(int argc, const char** argv) {
   amber_options.engine = options.engine;
   amber_options.spv_env = options.spv_env;
   amber_options.pipeline_create_only = options.pipeline_create_only;
+  amber_options.delegate = nullptr;
+
+  Delegate delegate;
```

Comment:
Move the delegate stuff up to 289, then you don't have to set the nullptr into amber_options.delegate you can just set the delegate